pkg/cmd/ui/validate.go

```diff
@@ -134,13 +134,18 @@ func validateAccessToken(input interface{}, serviceRepo string) error {
 // validateSealedSecretService validates to see if the sealed secret service is present in the correct namespace.
 func validateSealedSecretService(input interface{}, sealedSecretService *types.NamespacedName) error {
 	if s, ok := input.(string); ok {
-		sealedSecretService.Name = s
-		sealedSecretService.Namespace = EnterSealedSecretNamespace()
-		_, err := secrets.GetClusterPublicKey(*sealedSecretService)
+		client, err := utility.NewClient()
+		err = client.CheckIfNamespaceExists(s)
+		if errorType.IsNotFound(err) {
+			return fmt.Errorf("The namespace %s is not present on the cluster", s)
+		}
+		sealedSecretService.Namespace = s
+		sealedSecretService.Name = EnterSealedSecretService(sealedSecretService)
+		err = client.CheckIfSealedSecretsExists(*sealedSecretService)
+		if errorType.IsNotFound(err) {
+			return fmt.Errorf("The service %s is not present in the namespace %s", sealedSecretService.Name, sealedSecretService.Namespace)
+		}
 		if err != nil {
-			if compareError(err, sealedSecretService.Name) {
-				return fmt.Errorf("The given service %q is not installed in the right namespace %q", sealedSecretService.Name, sealedSecretService.Namespace)
-			}
 			return errors.New("sealed secrets could not be configured sucessfully")
```

Comment:
Perhaps we should include the actual error here, to make debugging easier?

Also, the error is misleading, we're not trying to configure sealed secrets?